package.json

```diff
@@ -15,6 +15,10 @@
   ],
   "dependencies": {},
   "devDependencies": {
+    "@babel/plugin-proposal-class-properties": "7.12.1",
+    "@babel/plugin-proposal-object-rest-spread": "7.12.1",
+    "@babel/preset-env": "7.12.11",
+    "@babel/preset-typescript": "7.12.7",
```

Comment:
This is unfortunately required to work around a bug, see https://github.com/storybookjs/storybook/issues/14197